### PR TITLE
Add paragraph to ecommerce for personalization and carousel

### PR DESCRIPTION
**Pull request by kul3r4**
This shows how to use `amp-list` as a src of a carousel

### CODE REVIEW

File: src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html [hunk 1]
```diff
@@ -142,6 +143,27 @@
       {{name}}: ${{price}}
     </template>
   </amp-list>
+
```
Comment (sebastianbenz) [inline src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html]: Can you add a heading

File: src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html [hunk 2]
```diff
@@ -142,6 +143,27 @@
...
+  <!-- E-Commerce websites tipically use carousel layout to display personalized content.
```
Comment (sebastianbenz) [inline src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html]: suggestion: 'It's common for e-commerce websites to display personalized content or recommendations in a carousel.'

File: src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html [hunk 3]
```diff
@@ -142,6 +143,27 @@
...
+  One way to address this is to have an `amp-list` as the src of `amp-carousel` and the amp-list endpoint
```
Comment (sebastianbenz) [inline src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html]: suggestion: One way to address/One way to implement this is to ...'

File: src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html [hunk 4]
```diff
@@ -142,6 +143,27 @@
...
+  needs to return a single entry in `items`, like this:
```
Comment (sebastianbenz) [inline src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html]: Why does it need to return a single entry items?

File: src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html (hunk 1 above)
Comment (kul3r4) [reply src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html]: done

File: src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html (hunk 2 above)
Comment (kul3r4) [reply src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html]: done

File: src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html (hunk 3 above)
Comment (kul3r4) [reply src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html]: done

File: src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html (hunk 4 above)
Comment (kul3r4) [reply src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html]: Find the explanation in [https://github.com/ampproject/amphtml/issues/4405#issuecomment-318204399](https://github.com/ampproject/amphtml/issues/4405#issuecomment-318204399)

**Comment by sebastianbenz** [comment]
You should add the explanation to the sample.

File: src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html [hunk 5]
```diff
@@ -147,7 +147,9 @@
   <!-- ## Display personalized content
 
   It's common for e-commerce websites to display personalized content or recommendations in a carousel.
-  One way to implement this is to have an `amp-list` as the src of `amp-carousel` and the amp-list endpoint
+  The `amp-mustache` default behaviour for rendering `amp-list` data is to cycle inside `items` objects and adding
```
Comment (sebastianbenz) [inline src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html]: suggestion: start a new sentence after objects.

File: src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html [hunk 6]
```diff
@@ -147,7 +147,9 @@
...
+  an `amp-carousel` inside the template this would make the template to render multiple carousels;
+  one way to implement this is to have an `amp-list` as the src of `amp-carousel` and the amp-list endpoint
```
Comment (sebastianbenz) [inline src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html]: * s/one way to implement this/one way to work around this
* I don't understand this bit "as the src of `amp-carousel` and the amp-list endpoint"

Comment (kul3r4) [reply src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html]: Ok, tried it again by moving the sentences around. It should be clearer now!

File: src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html [hunk 7]
```diff
@@ -142,6 +143,31 @@
       {{name}}: ${{price}}
     </template>
   </amp-list>
+
+  <!-- ## Display personalized content
+
+  It's common for e-commerce websites to display personalized content or recommendations in a carousel.
+  One way to implement this is to have an `amp-list` returning the content of an `amp-carousel`.
+  The `amp-mustache` default behaviour for rendering `amp-list` data is to cycle inside `items` objects; adding
+  an `amp-carousel` inside the template would make the template to render multiple carousels;
```
Comment (sebastianbenz) [inline src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html]: remove 'to' from to render

File: src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html [hunk 8]
```diff
@@ -142,6 +143,31 @@
...
+  one way to work around this is having the `amp-list` endpoint returning a single entry in `items`, like this:
```
Comment (sebastianbenz) [inline src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html]: s/returning/return

File: src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html (hunk 7 above)
Comment (kul3r4) [reply src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html]: Done!

File: src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html (hunk 8 above)
Comment (kul3r4) [reply src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html]: Done

**Pull request merged**